.github/workflows/_mac-build.yml

```diff
@@ -88,7 +91,7 @@ jobs:
 
       - name: Setup miniconda
         if: inputs.environment-file == ''
-        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
+        uses: huydhn/test-infra/.github/actions/setup-miniconda@macos-cleanup-after-job
```

Comment:
I assume is is just temporary :)

Reply:
Exactly, this is only temporary before I merge https://github.com/pytorch/test-infra/pull/1548, will sync the twos PR so that the gap is as small as possible